x/builder/types/bid_info.go

```diff
@@ -0,0 +1,12 @@
+package types
+
+import sdk "github.com/cosmos/cosmos-sdk/types"
+
+// BidInfo defines the information about a bid to the auction house.
```

Comment:
I thought that it would make more sense to keep concrete bid type in the `x/builder` module instead of the lane. Let me know what you think.